Tests/Data/ArraysTest.php

```diff
@@ -160,4 +160,42 @@ public function testArrayChangeCaseValue()
         $this->assertSame(['name' => 'ALEX'], Arrays::arrayChangeCaseValue($array, CASE_UPPER));
         $this->assertNotSame(['name' => 'Alex'], Arrays::arrayChangeCaseValue($array, CASE_UPPER));
     }
+
+    public function testRemoveDuplicates()
+    {
+        $dataSet1 = [
+            'users' => 
+            [
+                'id'       => 1,
+                'name'     => "Umer",
+                'username' => 'peter',
+            ],
+            [
+                'id'       => 2,
+                'name'     => "Umer",
+                'username' => 'umer01'
+            ],
+            [
+                'id'       => 3,
+                'name'     => "Peter Khot",
+                'username' => 'peter',
+            ]
+        ];
+        $dataSet2 = array("a" => "green", "red", "b" => "green", "blue", "red");
```

Comment:
Please use the short array syntax because of the consistency.

Reply:
It seems that the expected array will be changed the order in associative array.